### PR TITLE
Add released metrics to new focus areas

### PR DESCRIPTION
**Pull request by ccarterlandis**
This patch adds the metrics defined in the first formal metrics release
to their corresponding new focus areas. It also fixes a broken link in the
`focus_areas` README and updates the metric question template.

Please note that in this patch, **no actual metrics content has been changed**: barring typos or broken links, this patch solely reorganizes already released metrics.

Signed-off-by: Carter Landis <ccarterlandis@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +63,8 @@ Focus Area | Goal
 --- | ---
 [Code Development Activity](./code_development_activity.md) | Learn about the types and frequency of activities involved in developing code.
 [Code Development Efficiency](./code_development_efficiency.md) | Learn how efficiently activities around code development get resolved.
-[Code Development Process Quality](./code_development_quality.md) | Learning about the processes to improve/review quality that are used (for example: testing, code review, tagging issues, tagging a release, time to response, CII Badging).
+[Code Development Process Quality](./code_development_process_quality.md) | Learn about the processes to improve/review quality that are used (for example: testing, code review, tagging issues, tagging a release, time to response, CII Badging).
+[Issue Resolution](./issue_resolution.md) | Identify how effective the community is at addressing issues identified by community participants.
```
Comment (GeorgLink) [inline README.md]: Please double-check the links in the ./README.md 
I think they are missing the directory where the file is located.

File: focus_areas/templates/question_template.md [hunk 2]
```diff
@@ -1,37 +1,21 @@
-# [Area of Analysis]: [Focal Concern]
+## Description
```
Comment (GeorgLink) [inline focus_areas/templates/question_template.md]: I think we wanted to keep the first four lines of the old template to also have the Title and Question in the new template.

Please add Title and Question back into the template.

File: focus_areas/templates/question_template.md [hunk 3]
```diff
@@ -1,37 +1,21 @@
...
+### Tools providing the metric (optional)
+- formerly: "known or sample implementations"
+- Metric must be currently deployed/available, in contrast to a tool having the "potential" to provide the metric
+- provide direct link to implementation/documentation, if applicable
 
-- _This section provides a list of reasons a community wants to assess this question_
--
+### Data Collection Strategies (optional)
 
+## Success Metrics (optional)
```
Comment (GeorgLink) [inline focus_areas/templates/question_template.md]: Success Metrics should also be a sub-heading under Implementation. 

Please adjust the heading level.

**Comment by ccarterlandis** [comment]
@GeorgLink I believe I've correctly fixed the formatting/ link errors, please let me know if I missed anything. Thank you for catching them!

File: focus_areas/templates/question_template.md [hunk 4]
```diff
@@ -1,37 +1,25 @@
-# [Area of Analysis]: [Focal Concern]
+## Title
 
-**Question:** placeholder for a variety of channels (choice?)
+## Question
```
Comment (GeorgLink) [inline focus_areas/templates/question_template.md]: Line 3: "Question" should not be a sub-heading but a bolded word.

File: focus_areas/templates/question_template.md [hunk 5]
```diff
@@ -1,37 +1,25 @@
-# [Area of Analysis]: [Focal Concern]
+## Title
```
Comment (GeorgLink) [inline focus_areas/templates/question_template.md]: Title should be heading 1 not heading 2 (one `#`, not two `##`)

**Comment by GeorgLink** [comment]
On a second thought, how about reusing the metrics-template we already have in the chaoss/metrics repo? 

https://github.com/chaoss/metrics/blob/master/activity-metrics/metric-template.md

**Comment by germonprez** [comment]
This template (in the metrics repo) needs to be updated. I'll do that here shortly.

**Comment by ccarterlandis** [comment]
@GeorgLink I've updated the format with your suggestions. As a side note, the force-push was from before I realized that @germonprez's comment about the template was referring to the `chaoss/metrics` template, not the one in this repo. Whoops! 😄

**Comment by GeorgLink** [comment]
Okay, thanks!

**Pull request merged**